config/routes.rb

```diff
@@ -150,6 +150,7 @@ def external_redirect(path_prefix, target)
 
     resources :world_locations, path: 'world', only: [:index, :show], localised: true do
       resources :world_location_news, path: 'news', only: [:index]
+      resources :worldwide_organisations, path: '', only: [:show], to: 'worldwide_organisations#show_b_variant'
```

Comment:
It suppresses `/worldwide_organisations` in the url so we get a url in the form of `/government/world/<location>/<organisation>` rather than `/government/world/<location>/worldwide_organisations/<organisation>`

Not sure if there's a better way to do that?